vocabularies/Offline.xml

```diff
@@ -37,20 +34,42 @@ Please refer to the [Offline OData product home](https://help.sap.com/doc/f53c64
           </Record>
         </Collection>
       </Annotation>
-
-      <Term Name="ClientOnly" Type="Core.Tag" DefaultValue="true" Nullable="false" AppliesTo="EntityType EntitySet EnumType ComplexType TypeDefinition">
+      <Term Name="ClientOnly" Type="Offline.ClientOnlyType" Nullable="false" AppliesTo="EntityType EntitySet EnumType ComplexType TypeDefinition">
         <Annotation Term="Common.Experimental" />
         <Annotation Term="Core.Description" String="The annotated model element exists only on client devices"/>
          <Annotation Term="Core.LongDescription">
-           <String>Occasionally a customer will want to store additional “client-only”
-           entities in the same DB. The easiest way to accommodate this is for the client 
-           to extend the backend-returned $metadata (upon which the local persistence is 
-           based) and just to mark some entities as client-only so that the client won’t
-           attempt to upload any changes that are made to them locally.
-           The service as implemented on the server is described by the CSDL document if
-           all elements tagged as `Offline.ClientOnly` are removed together with all subsequently dangling references.</String>
-        </Annotation>        
+           <String>
+Occasionally a customer will want to store additional “client-only” entities in the same database.
+The easiest way to accommodate this is for the client to extend the backend-returned $metadata
+(upon which the local persistence is based) and to mark some entities as client-only so that the
+client won’t attempt to upload any changes that are made to them locally.
+The service as implemented on the server is described by the CSDL document if all elements tagged
+as `Offline.ClientOnly` are removed together with all subsequently dangling references.
```

Comment:
```suggestion
The service as implemented on the server is described by the CSDL document if all elements annotated
with `Offline.ClientOnly` are removed together with all subsequently dangling references.
```
Because `ClientOnly` is no longer a tag.